src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -434,6 +436,16 @@ CompositeOrderDynamicDataPopulateService combinedPopulateService(CompositeOrderR
     return new CombinedOrderDataPopulateService(compositeOrderRetrieveHolderBuilder, populateServices);
   }
 
+  @Bean
+  RoutingListsService routingListService(RestClient restClient, UserService userService) {
```

Comment:
propose to name service as RoutingListService, because most of services we have in singular
![image](https://github.com/folio-org/mod-orders/assets/25097693/b54e05da-254e-4899-912a-d1c357c82e13)

Reply:
changed